src/commonMain/kotlin/character/classes/druid/specs/Boomkin.kt

```diff
@@ -0,0 +1,27 @@
+package character.classes.druid.specs
+
+import character.Spec
+import character.SpecEpDelta
+
+/**
+ */
+class Boomkin : Spec() {
+    override val name: String = "Boomkin"
+    override val epBaseStat: SpecEpDelta = spellPowerBase
+    override val epStatDeltas: List<SpecEpDelta> = defaultCasterDeltas
+
+    override fun redSocketEp(deltas: Map<String, Double>): Double {
```

Comment:
I wasn't sure what the purpose of these were actually. I just copied it straight from the warlock destruction spec.

Reply:
These are for EP calculation - each socket should have an implementation that reasonably estimates which stat would be put in that socket.  

I believe boomies would generally use spelldamage in every socket, so the yellow could probably use that as a basis as well.